### ISSUE TITLE
Use deno-shell for a cross platform shell

### ISSUE THREAD

**Issue by sigmaSd** [c4spar/deno-dzx#85]
This have the advantage of having a cross platform shell out of the box , instead of using ffi you can actually use wasm see https://github.com/dsherret/ax which does exactly this.

**Comment by c4spar** [comment]
Yeah, that would definitely be nice and i already had this in mind. I mentioned this idea a few months ago in https://github.com/denoland/deno/issues/14068#issuecomment-1075756137 and also started testing some simple bindings for `deno_task_shell` in https://github.com/c4spar/deno-shell.
